web/cobrands/highways/assets.js

```diff
@@ -126,8 +126,8 @@ function non_he_selected() {
 }
 
 function add_highways_warning(road_name) {
-  var $warning = $('<div class="box-warning" id="highways"><p>It looks like you clicked on the <strong>' + road_name + '</strong> which is managed by <strong>Highways England</strong>. ' +
-                   'Does your report concern something on this road, or somewhere else (e.g a road crossing it)?<p></div>');
+  var $warning = $('<div data-page-name="highwaysengland" class="js-reporting-page js-reporting-page--active"><div class="box-warning" id="highways"><p>It looks like you clicked on the <strong>' + road_name + '</strong> which is managed by <strong>Highways England</strong>. ' +
+                   'Does your report concern something on this road, or somewhere else (e.g a road crossing it)?<p></div></div>');
   var $radios = $('<p class="segmented-control segmented-control--radio"></p>');
```

Comment:
This isn't being cleared properly if I click on a Highways England road, and then click off it. The message goes but the segmented controller remains. I guess because it's not removing the page?

Reply:
Yes, this was me wrapping the old div, but then leaving the code that was removing only the inner div.